app/content/pub-sub.md

```diff
@@ -33,5 +33,5 @@ This console displays all pub/sub notifications that your application processes
 <div id="checkbox-filter-container"></div>
 <br>
 <div id="notificationsContainer">
-    <div id="notificationsLog" style="overflow:scroll; max-height:500px;"></div>
+    <div id="notificationsLog"></div>
```

Comment:
This all looks like a great fix, but why uncap the scroll here too? Now the list of notifications breaks out of the page, which looks pretty bad visually.